### PR TITLE
Fix switch to tty6 failed in orphaned_packages_check

### PR DESCRIPTION
**Pull request by hjluo**
Sometimes the select_console can't switch to tty6 in orphaned_packages_check while migration from SLES15 to  SLES15-SP2. To fix it, we need to move orphaned_packages_check after post_migration.
We also have another ticket https://progress.opensuse.org/issues/48110 for this.

- Related ticket: 
    https://progress.opensuse.org/issues/57281
    https://progress.opensuse.org/issues/48110
- Needles: N/A
- Verification run: 
  https://openqa.suse.de/tests/3769433#step/orphaned_packages_check/6
  https://openqa.suse.de/tests/3769437#step/orphaned_packages_check/6

### CODE REVIEW

**Comment by okurz** [comment]
The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages. Also please start the git commit message subject line with a capital letter and remove the dot at the end of line.

**Comment by hjluo** [comment]
> The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages. Also please start the git commit message subject line with a capital letter and remove the dot at the end of line.

Changed.

**Comment by okurz** [comment]
The *commit message* is still missing the details. Please keep in mind that the git commit message is the one that is preserved in the git history, the github pull request description is not. Having a good commit message can be very helpful for bug investigation and being able to follow the history and exactly the reason *why* things have been changed. I can recommend to use the CLI tool "hub" to create pull requests from the command line without even needing to interact with the browser. I use https://github.com/okurz/scripts/blob/master/git-pr-last to create a single-commit pull request or https://github.com/okurz/scripts/blob/master/git-pr-more for a pull request consisting of multiple commits.

File: tests/console/orphaned_packages_check.pm [hunk 1]
```diff
@@ -38,8 +38,8 @@ sub run {
 sub post_fail_hook {
     my $self = shift;
 
-    $self->export_logs();
-    upload_logs '/tmp/orphaned.log';
+    $self->SUPER::post_fail_hook;
+    $self->upload_packagekit_logs;
```
Comment (okurz) [inline tests/console/orphaned_packages_check.pm]: this change is unrelated to the "switch to tty6". Also, why use the "packagekit_logs"? Also you are loosing the log of the orphaned packages.

Comment (hjluo) [reply tests/console/orphaned_packages_check.pm]: I've just followed your suggestion on the ticket, what do you want to change here? any suggestion?

**Comment by hjluo** [comment]
After discussed with Lemon, we'd like to close this PR cause with his PR 9270, which to remove orphaned_package_check from online migration to regression tests. there is no need to fix it in online
migration.

For further information please refer to the ticket https://progress.opensuse.org/issues/61892.
thanks for the review.
